src/main/java/devcourse/board/api/PostApiController.java

```diff
@@ -0,0 +1,54 @@
+package devcourse.board.api;
+
+import devcourse.board.domain.post.PostService;
+import devcourse.board.domain.post.model.PostRequest;
+import devcourse.board.domain.post.model.PostResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/posts")
+public class PostApiController {
+
+    private final PostService postService;
+
+    public PostApiController(PostService postService) {
+        this.postService = postService;
+    }
+
+    @GetMapping("/{postId}")
+    public ResponseEntity<ApiResponse<PostResponse>> getPost(
+            @PathVariable Long postId
+    ) {
```

Comment:
```suggestion
    public ResponseEntity<PostResponse> getPost(
            @PathVariable Long postId
    ) {
```

ApiResponse에 PostResponse까지 적으니 어색하고 길어져서 부자연스럽네요 🤔